common/emitter/x86emitter.cpp

```diff
@@ -151,17 +151,12 @@ const xRegister8
     ah(4), ch(5),
     dh(6), bh(7);
 
-#if defined(_WIN32) || !defined(__M_X86_64)
+#if defined(_WIN32)
 const xAddressReg
     arg1reg = rcx,
     arg2reg = rdx,
-#ifdef __M_X86_64
     arg3reg = r8,
     arg4reg = r9,
```

Comment:
Not quite convinced the logic here is right, it should probably be
if win32
rcx, rdx
else
r8, r9